src/Alert/Alert.js

```diff
@@ -0,0 +1,25 @@
+import { toast } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
+
+export const successAlert = () => toast.success("Success!", {
+        position: toast.POSITION.BOTTOM_CENTER,
+        hideProgressBar: true,
+        autoClose: 3000
+})
+
+export const errorAlert = () => toast.error("User not found!", {
+        position: toast.POSITION.BOTTOM_CENTER,
+        hideProgressBar: true,
+        autoClose: 3000
+})
```

Comment:
This will probably need to wait for a future iteration, because offline functionality will change a lot of this, but I'm wondering if we should replace all non 404 errors with toasts? So if we get a 500 error a toast tells you that something went wrong.

Reply:
Yeah Boone, I agree I was thinking this too that it would be nice to have all our error handling alerts be toasts.